app/views/friendships/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with(model: friendship, local: true) do |form| %>
+  <% if friendship.errors.any? %>
```

Comment:
Hi, it would really help you if you remove logic from all view files. 
A better approach if you want a loop inside a conditional statement is to put the conditional in your helper files and put the loops in partials that you call in the helper. That way, you can just call the helper in the view. This makes your code cleaner and easier to test. 😃 😃